data/docs/userguide/logs_quick_filters.mdx

```diff
@@ -0,0 +1,82 @@
+---
+date: 2024-12-02
+title: Logs Quick Filters
+id: logs_quick_filters
+---
+
+The Logs Explorer comes with quick filters which allows filtering of logs based on key attributes making it easier to analyze and debug applications.
+Here's a detailed explaination of the available filters and how to use them -
+
+<figure data-zoomable align="center">
+  <img
+    src="/img/docs/logs-management/ui/logs-quick-filters.webp"
+    alt="Quick Filters in Logs Explorer"
+  />
+  <figcaption>
+    <i> Quick Filters in Logs Explorer </i>
+  </figcaption>
+</figure>
+
+### Available Filters
+
```

Comment:
There are multiple filtering options available, with each filter mapped to a specific [OpenTelemetry (OTEL) attribute](https://opentelemetry.io/docs/specs/otel/common/#attribute). Here’s a list of the available filters: